packages/next/src/build/index.ts

```diff
@@ -1010,8 +1011,17 @@ export default async function build(
           ignore: [] as string[],
         }))
 
+      let binding = (await loadBindings()) as any
+
+      async function runTurboBuild() {
```

Comment:
Nit: To make it more consistent with `webpackBuild`:

```suggestion
      async function turbopackBuild() {
```